src/dmd/todt.d

```diff
@@ -165,7 +165,7 @@ extern (C++) void Initializer_toDt(Initializer init, ref DtBuilder dtb)
                 }
                 else if (ai.dim > tadim)
                 {
-                    error(ai.loc, "too many initializers, %d, for array[%d]", ai.dim, tadim);
+                    error(ai.loc, "too many initializers, %u, for array[%zu]", ai.dim, tadim);
```

Comment:
`%u` for `dim ?

Reply:
Nope:
https://github.com/dlang/dmd/blob/b4744b28e21cbe28a98650417492a8a79367095f/src/dmd/init.d#L164-L168

Reply:
Sneaky

Reply:
Was touching that line anyway, better fix it now than later